catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +46,21 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString = (String) arguments.get("columnOrder");
+    List<String> attributes =
+        new ArrayList<String>(Arrays.asList((attributeString).split(",")));
+    List<AttributeDescriptor> allAttributes =
+        new ArrayList<AttributeDescriptor>(metacard.getMetacardType().getAttributeDescriptors());
+    List<AttributeDescriptor> descriptors =
+        new ArrayList<AttributeDescriptor>(
+            attributes == null || attributes.isEmpty()
```

Comment:
✏️ Same comment as the other PR, could use CollectionUtils.isEmpty(attributes) here